### DISCUSSION TITLE
Trausti villtist til Húsavíkur - 2023/5/1

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#15] [vefsida-ummæli]
# Trausti villtist til Húsavíkur - 2023/5/1

Trausti fer til Norðausturlands

https://www.biskupsreglan.com/verkefni/trausti-fer-til-husavikur

**Comment by MatthiasGudni** [comment]
Þetta er ekkert eðlilega sorglegt, ég vona að manngreyið ratar heim á næstunni. ❤️

**Comment by TJ478** [comment]
Ég komst heim.
